lib/rules/indent.js

```diff
@@ -0,0 +1,478 @@
+/*eslint no-use-before-define:[2, "nofunc"]*/
+/**
+ * @fileoverview This option sets a specific tab width for your code
+ * @author Dmitriy Shekhovtsov
+ * @author Mike Sherov <mikesherov>
+ * @copyright 2015 Dmitriy Shekhovtsov. All rights reserved.
```

Comment:
Remove `@author` for Mike, and instead add his copyright under yours:

`@copyright 2014 Mike Sherov.`

We also should mention in this fileoverview comment that this file has been ported and modified from JSCS.